scripts/hid_configurator/README.rst

```diff
@@ -44,6 +44,7 @@ Complete the following steps:
 
 1. Download the HIDAPI library from `HIDAPI releases`_.
    Use the bundled DLL or build it according to instructions in `HIDAPI library`_.
+#. Copy the proper version of :file:`hidapi.dll` file to the directory where :file:`python.exe` is located.
```

Comment:
`It may be needed to copy...` ?

Reply:
I think you must do it, so simply `Copy` is better here. Right after building/downloading the file will not be properly located, so the user will have to copy/move it.

Reply:
Normally such things are handled by setting up path to linked libraries. I have spent no time checking how to do that on Windows but it would be odd if you won't be able to tell OS where to look for a lib. On Linux you can do so with LD_LIBRARY_PATH.
Did a quick look - 2 minutes in google give no definitive answer so maybe it is more difficult to do on Win.

Reply:
What do you mean by `proper version` here?